### ISSUE TITLE
Asset prefix not working for stylesheets

### ISSUE THREAD

**Issue by geekyme** [zeit/next.js#5673]
# Bug report

## Describe the bug

Our app is deployed on a subpath: `http://myapp.com/abc`. We observe that even if we set assetPrefix to `http://myapp.com/abc`, urls in our stylesheets are still not automatically resolved to load with the prefix. 

## To Reproduce

```
 @font-face {
   src: url("/static/assets/Fonts/icomoon.eot?bocr88");
   font-weight: normal;
   font-style: normal;
 }
```

the above asset will be loaded from `http://myapp.com/static/assets/Fonts/icomoon.eot?bocr88`, which is unexpected.

## Expected behavior

should load from `http://myapp.com/abc/static/assets/Fonts/icomoon.eot?bocr88`

## System information
 - OS: macOS
 - Browser: all browsers
 - Version of Next.js: 7.0.2

## Additional context

The reason why our app has to be deployed on a subpath is because we have an nginx sitting in front that will direct traffic on different path to different applications. There is no way to change to using a subdomain which will simplify things.

**Comment by timneutkens** [comment]
This is not a bug in Next.js, and there's not much we can do on the Next.js side to solve this, you'll probably want to add loaders for these file types or something, as Next.js doesn't support css import out of the box, it also doesn't handle resolving file urls.

**Comment by geekyme** [comment]
What about nextjs <Link />? If an app is on http://myapp.com/abc, will clicking a nextjs <Link /> to /one go to http://myapp.com/abc/one with assetPrefix?


On 14 Nov 2018, at 7:58 PM, Tim Neutkens <notifications@github.com<mailto:notifications@github.com>> wrote:


Closed #5673<https://github.com/zeit/next.js/issues/5673>.
